.travis.yml

```diff
@@ -107,6 +107,10 @@ script:
       cp appdir/usr/bin/images/treesheets.svg appdir/usr/share/icons/hicolor/scalable/apps/ # FIXME
       ( cd appdir ; ln -s usr/bin/docs/ usr/bin/examples/ usr/bin/images/ usr/bin/scripts/ usr/bin/translations/ . )
       sed -i -e 's|1011|1002|g' appdir/usr/bin/treesheets # https://github.com/aardappel/treesheets/issues/130#issuecomment-528075693
+      # Workaround to increase compatibility with older systems; see https://github.com/darealshinji/AppImageKit-checkrt for details
```

Comment:
Live'd be much easier with the linuxdeploy, where we have a plugin for this kind of stuff.

There's much custom weird undocumented script code in this file...

Reply:
Possibly. But this I had available for copy-and-paste, and it is working for me.